agir/groups/components/groupComponents/GroupCard.js

```diff
@@ -56,6 +56,14 @@ const DiscountCodesSection = styled.section`
   }
 `;
 
+const StyledGroupButton = styled(Button)`
+  height: 48px;
+  font-size: 16px;
+  font-weight: 500;
+  border-radius: 0.5rem;
```

Comment:
On a une variable maintenant pour la nouvelle valeur de border-radius :
```suggestion
  border-radius: ${style.borderRadius};
```